### DISCUSSION TITLE
[Bulk Data question] Missing judges from dockets

### DISCUSSION THREAD

**Discussion by arharvard** [freelawproject/courtlistener#2817] [Q&A]
Hello,

In some districts I encounter missing judges (either missing "assigned to str" or "assigned to id"), while looking up the docket I do see a judge in that docket. Can I find the judge-case in another file perhaps? 

Thanks

**Comment by mlissner** [comment]
Can you give an example or two, please?

**Comment by arharvard** [comment]
Sure, see images below for example for (1) ilsb (Illinois southern bankruptcy) dockets: 16-31833,17-60198,18-31571. (2) ksb (Kansas bankruptcy) dockets: 09-10930,10-10182,12-10093.

Also, I have attached sample dockets from SCB with missing judges, but when looking up the dockets, there is a judge listed.

Any help here would be highly appriciated.

[sample_missing_judges.xlsx](https://github.com/freelawproject/courtlistener/files/11735951/sample_missing_judges.xlsx)
![Screenshot 2023-06-13 at 09 52 48](https://github.com/freelawproject/courtlistener/assets/127220811/23b67742-a780-4f51-a0d0-d37924ff8920)
![Screenshot 2023-06-13 at 09 52 32](https://github.com/freelawproject/courtlistener/assets/127220811/b6345800-1f2a-4b79-a8d4-c1b983d74972)

**Comment by flooie** [comment]
Hi @arharvard 

Unfortunately that information is not contained somewhere else.  But we think we found a bug in either the parsing or the saving mechanism for bankruptcy (and maybe district) cases.  We should be able to fix that and re-extract the data of the judge into the dockets though.  

An issue should be created this week that you can follow along with here in this repository.  Thanks.

@mlissner

**Reply by arharvard** [reply]
Excellent, thank you!

**Comment by flooie** [comment]
@arharvard here is the link to the issue https://github.com/freelawproject/courtlistener/issues/2819
